app/editor/src/components/form/textarea/TextArea.tsx

```diff
@@ -1,3 +1,5 @@
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
I don't think these will work as they should be from the `react-icons`.

Reply:
These seem to be working based on the testing here, in the meantime, they were already in the editor app's package.json file before and currently being used in other editor forms as well, so maybe we should have a chat about this to confirm when you have time? Thanks

Reply:
Ah, we'll remove them in another PR then.